### ISSUE TITLE
Freeze child regressors of stacking estimators

### ISSUE THREAD

**Issue by kota7** [rasbt/mlxtend#439]
Is it possible to "freeze" child regressors of `StackingRegressor` or `StackingClassifier` so that only meta regressor is fitted?  In some situations I train a lot of models separately, and look for a way I can simply ensemble them without fine tuning.
If this feature is currently not available, would this be good one to have?

**Comment by rasbt** [comment]
Currently, that's not supported. Only the `EnsembleVoteClassifier` has currently a `refit` parameter to specify this (i.e., if it is set to `False`, it assumes the classifiers are already fit).

However, I think it should be easy to provide this functionality in the `Stacking*` classes as well. I would welcome a PR for this if you like to do the modification.

**Comment by kota7** [comment]
@rasbt I will take a look at it.  Do you have preference for the name of option?  It seems refit is already there to mean different thing.

**Comment by rasbt** [comment]
Good point. I replaced `refit` with `use_clones` in the stacking classifiers but forgot to do that for the stacking regressors. `refit` is a scikit-learn API term, but in the context of model ensembles it's highly confusing, which is why I decided to rename that. It should be done for the regressors, too, I'd say -- at least for consistency.

Top off my head, I don't have any really good term in mind for *this* feature. But I think "prefit" would be a good one, i.e., `prefit=True` assumes that models were prefit and don't have to be refit on the training data. The default (i.e., current behavior) should then be `prefit=False`. What do you think?

**Comment by kota7** [comment]
I see...  In sklearn terminology, `refit` means "make clones" so the update of internal estimators is isolated.

`prefit`would be good for this feature.  I will work on this extension.

**Comment by evbomac** [comment]
`prefit` would be very helpful when the child regressors are quite large / slow to train, or if you wanted to stack regressors with different features.  Would this be particularly hard to implement?

**Comment by rasbt** [comment]
I think for both `StackingRegressor` and `StackingClassifier`, adding a `prefit` param would be a nice edition (I think for the StackingCVClassifier, it would not work though). If someone wants to add it via a PR, I am all for it
